Homework/Develop/index.html

```diff
@@ -22,61 +23,69 @@ <h1>Hori<span class="seo">seo</span>n</h1>
                     <a href="#social-media-marketing">Social Media Marketing</a>
                 </li>
             </ul>
-        </div>
+
+        </nav>
+    </header>
+
+    <div class="hero">
+
     </div>
-    <div class="hero"></div>
-    <div class="content">
-        <div class="search-engine-optimization">
-            <img src="./assets/images/search-engine-optimization.jpg" class="float-left" />
+    <Main>
+        <div id="search-engine-optimization" class="service-section">
+            <img src="./assets/images/search-engine-optimization.jpg" alt="Notepad on table with SEO ideas drawn out" class="float-left" />
             <h2>Search Engine Optimization</h2>
             <p>
                 The dominance of mobile internet use means that users are searching for the right business as they travel, shop, or sit on their couch at home. Search Engine Optimization (SEO) allows you to increase your visibility and find the right customers for your business.
             </p>
         </div>
-        <div id="online-reputation-management" class="online-reputation-management">
-            <img src="./assets/images/online-reputation-management.jpg" class="float-right" />
+        <div id="online-reputation-management" class="service-section">
+            <img src="./assets/images/online-reputation-management.jpg" alt="laptop open with an illustration of charts and a graph showing and upward trend titled reputation" class="float-right" />
             <h2>Online Reputation Management</h2>
             <p>
                 The web is full of opinions, and some of these can be negative. Social media allows anyone with an internet connection to say whatever they want about your business. Online Reputation Management gives you the control over what potential customers see when they search for your business.
             </p>
         </div>
-        <div id="social-media-marketing" class="social-media-marketing">
-            <img src="./assets/images/social-media-marketing.jpg" class="float-left" />
+        <div id="social-media-marketing" class="service-section">
+            <img src="./assets/images/social-media-marketing.jpg" alt="group of people around table with different social media terms spread out on the table" class="float-left" />
             <h2>Social Media Marketing</h2>
             <p>
                 Social media continues to have a sizable influence on buying habits. Social media marketing helps you determine which platforms are suited to your brand, using analytics to find the right markets and increase your lead generation.
             </p>
         </div>
     </div>
-    <div class="benefits">
-        <div class="benefit-lead">
+    </Main>
+
+    <aside>
+        <div class="benefit">
```

Comment:
Thing for you to research on: Can we use `section` tags within an `aside` tag? Are there other ways in which we can list out the 3 benefits within the `aside` tag? If you change anything please check the HTML page to see if there are any breaking changes.

Reply:
Changed the 'divs' to 'ul' 'li' to identify them better. updated the CSS file to reflect these changes here 84fbd8554660714265b24b0b6ae06cc5bd5d3a16